### PR TITLE
add xdpyinfo

### PR DESCRIPTION
**Pull request by satmandu**
Good for checking to see if x server is working.

Works properly:
- [x] x86_64

### CODE REVIEW

File: packages/xdpyinfo.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  depends_on 'xorg_proto' 
+
+        def self.build
+                system "./configure #{CREW_OPTIONS}"
+                system 'make'
+        end
+        
+        def self.install
+                system "make DESTDIR=#{CREW_DEST_DIR} install"
+        end
```
Comment (uberhacker) [inline packages/xdpyinfo.rb:29]: Why so much indentation?

**Comment by uberhacker** [comment]
Thank you for the new package @satmandu!

File: packages/xdpyinfo.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Adapted from Arch Linux xorg-xdpyinfo PKGBUILD at:
+# https://github.com/archlinux/svntogit-packages/raw/packages/xorg-xdpyinfo/trunk/PKGBUILD
+
+require 'package'
+
+class Xorg_xdpyinfo < Package
+  description 'Display information utility for X'
+  version '1.3.2-4'
```
Comment (uberhacker) [inline packages/xdpyinfo.rb:8]: Why do we need `-4`?

Comment (satmandu) [reply packages/xdpyinfo.rb:8]: Sorry that was a straight import from the arch pkgbuild, and I hadn't reset the -4.

File: packages/xdpyinfo.rb (hunk 1 above)
Comment (satmandu) [reply packages/xdpyinfo.rb:29]: my geany indents are getting reset... fixing that.

**Comment by satmandu** [comment]
fixed!

**Pull request merged**